apps/central-scan/frontend/package.json

```diff
@@ -47,9 +47,6 @@
       "last 1 safari version"
     ]
   },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
```

Comment:
this setting in `package.json` is lower in priority than `.eslintrc` so it, and the `eslint-config-react-app` package, were not doing anything and could be removed.